From 909b295a2ea57fd23060b67e760f77d8f92f5785 Mon Sep 17 00:00:00 2001
From: Peter Swain <peter.swain@cavium.com>
Date: Fri, 29 Aug 2014 12:14:02 -0700
Subject: [PATCH 140/202] MIPS: OCTEON: cpu_state not just for _HOTPLUG

previous commit "Remove oct-app-ctl hooks from SMP handlers"
placed cpu_state definition under CONFIG_CPU_HOTPLUG,
but it's also used in non-hotplug smp startup

Signed-off-by: Peter Swain <peter.swain@cavium.com>
[Original patch taken from Cavium SDK 3.1.1-544]
Signed-off-by: Bin Jiang <bin.jiang@windriver.com>
---
 arch/mips/cavium-octeon/smp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/mips/cavium-octeon/smp.c b/arch/mips/cavium-octeon/smp.c
index 77f2a7e..d620ab4 100644
--- a/arch/mips/cavium-octeon/smp.c
+++ b/arch/mips/cavium-octeon/smp.c
@@ -28,10 +28,10 @@ __cpuinitdata unsigned long octeon_processor_gp;
 static void **octeon_bootvector;
 static void *octeon_hotplug_entry_raw;
 extern asmlinkage void octeon_hotplug_entry(void);
+#endif
 
 /* State of each CPU. */
 DEFINE_PER_CPU(int, cpu_state);
-#endif
 
 static void octeon_icache_flush(void)
 {
-- 
1.8.2.1

